### ISSUE TITLE
cannot connect through xrdp

### ISSUE THREAD

**Issue by robertwenquan** [codemaze-fun/noi-linux#22]
![image](https://user-images.githubusercontent.com/8893623/223004544-fe9605f2-29a0-48d4-9895-7feeabae0e50.png)

auth does not work.

docker console throws errors

```
xrdp[373]: (373)(140443917244224)[ERROR] xrdp_wm_log_msg: Error connecting to sesman: 127.0.0.1 port: 3350

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 46910)

xrdp[373]: (373)(140443917244224)[DEBUG] return value from xrdp_mm_connect 1

xrdp[373]: (373)(140443917244224)[DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 47188)

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 47484)

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 34814)

xrdp[373]: (373)(140443917244224)[ERROR] xrdp_wm_log_msg: Error connecting to sesman: 127.0.0.1 port: 3350

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 35050)

xrdp[373]: (373)(140443917244224)[DEBUG] return value from xrdp_mm_connect 1

xrdp[373]: (373)(140443917244224)[DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 44256)

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 44542)

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 44806)

xrdp[373]: (373)(140443917244224)[ERROR] xrdp_wm_log_msg: Error connecting to sesman: 127.0.0.1 port: 3350

xrdp[373]: (373)(140443917244224)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 35962)

xrdp[373]: (373)(140443917244224)[DEBUG] return value from xrdp_mm_connect 1
```

**Comment by robertwenquan** [comment]
xorg connection error

```
xrdp[630]: (630)(139628358440768)[INFO ] TLS connection established from ::ffff:172.17.0.1 port 40628: TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384

xrdp[630]: (630)(139628358440768)[DEBUG] xrdp_00000276_wm_login_mode_event_00000001

xrdp[630]: (630)(139628358440768)[INFO ] Cannot find keymap file /etc/xrdp/km-00000804.ini

xrdp[630]: (630)(139628358440768)[INFO ] Cannot find keymap file /etc/xrdp/km-00000804.ini

xrdp[630]: (630)(139628358440768)[INFO ] Loading keymap file /etc/xrdp/km-00000409.ini

xrdp[630]: (630)(139628358440768)[WARN ] local keymap file for 0x00000804 found and doesn't match built in keymap, using local keymap file

xrdp[630]: (630)(139628358440768)[DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350

xrdp[630]: (630)(139628358440768)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 58300)





xrdp[630]: (630)(139628358440768)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 47828)

xrdp[630]: (630)(139628358440768)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 48120)

xrdp[630]: (630)(139628358440768)[ERROR] xrdp_wm_log_msg: Error connecting to sesman: 127.0.0.1 port: 3350

xrdp[630]: (630)(139628358440768)[DEBUG] Closed socket 17 (AF_INET6 ::ffff:127.0.0.1 port 53594)

xrdp[630]: (630)(139628358440768)[DEBUG] return value from xrdp_mm_connect 1
```

**Comment by robertwenquan** [comment]
after launched `xrdp-sesman`, 
the connection went through one more step, and threw this error

![image](https://user-images.githubusercontent.com/8893623/223009015-ef25ef1d-695b-4517-8ee0-091d288e3858.png)

**Comment by robertwenquan** [comment]
Xorg connection times out

```
xrdp-sesman[1107]: pam_unix(xrdp-sesman:session): session opened for user abc by (uid=0)
xrdp-sesman[1107]: pam_systemd(xrdp-sesman:session): Failed to connect to system bus: No such file or directory
xrdp-sesman[1107]: (1107)(140184148723264)[DEBUG] Closed socket 6 (AF_INET6 ::ffff:127.0.0.1 port 3350)

xrdp-sesman[1107]: (1107)(140184148723264)[DEBUG] Closed socket 7 (AF_INET6 ::ffff:127.0.0.1 port 3350)

xrdp[1097]: (1097)(140070732273472)[DEBUG] Closed socket 18 (AF_UNIX)

xrdp[1097]: (1097)(140070732273472)[DEBUG] Closed socket 18 (AF_UNIX)

xrdp-sesman[1108]: (1108)(140184148723264)[ERROR] X server for display 10 startup timeout
xrdp-sesman[1107]: (1107)(140184148723264)[ERROR] X server for display 10 startup timeout


xrdp-sesman[1107]: (1107)(140184148723264)[CORE ] waiting for window manager (pid 1108) to exit

xrdp-sesman[1107]: (1107)(140184148723264)[CORE ] window manager (pid 1108) did exit, cleaning up session

xrdp-sesman[1107]: (1107)(140184148723264)[INFO ] calling auth_stop_session and auth_end from pid 1107

xrdp-sesman[1107]: pam_unix(xrdp-sesman:session): session closed for user abc
xrdp-sesman[1107]: (1107)(140184148723264)[DEBUG] cleanup_sockets:

xrdp-sesman[647]: (647)(140184148723264)[INFO ] ++ terminated session:  username abc, display :10.0, session_pid 1107, ip ::ffff:172.17.0.1:58216 - socket: 11

xrdp[1097]: (1097)(140070732273472)[DEBUG] Closed socket 18 (AF_UNIX)

xrdp[1097]: (1097)(140070732273472)[DEBUG] Closed socket 18 (AF_UNIX)

```

**Comment by robertwenquan** [comment]
after fix

![image](https://user-images.githubusercontent.com/8893623/223015554-7b68d4c2-fe57-4965-8565-45c6bfe9ac68.png)


![image](https://user-images.githubusercontent.com/8893623/223015514-073f63c7-b469-4b5d-bdf4-f0e717fc7ba6.png)
